cbrkit/loaders.py

```diff
@@ -76,10 +76,34 @@ def __len__(self) -> int:
 
 
 def dataframe(df: DataFrame) -> Casebase[Any, pd.Series]:
+    """Converts a pandas DataFrame into a Casebase.
+
+    Args:
+        df: pandas DataFrame.
+
+    Returns:
+        Returns a Casebase as a DataFrameCasebase.
+
+    Examples:
+        >>> df = pd.read_csv(casebase_file)
```

Comment:
This throws an error when running pytest since `casebase_file` is not defined. You could use a real csv like `cars-1k.csv` form the data folder instead.